src/foam/u2/view/LoginView.js

```diff
@@ -143,6 +143,13 @@ foam.CLASS({
   ],
 
   methods: [
+    {
+      name: 'detach',
+      code: function() {
+        this.SUPER();
+        this.data.detach();
```

Comment:
Is this needed?

Reply:
Or harmful?

Reply:
It is needed because those models sub to listeners and those listeners never get detached when the view is destroyed. The SignIn SignUp models are one off anyway so shouldn't be harmful